### PR TITLE
Set User-Agent Header with ripple-lib/{version}

### PR DESCRIPTION
**Pull request by lumberj**
- This would be helpful for us to analyze usage of ripple-lib for anyone
  using the public rippled

See: https://github.com/websockets/ws/blob/master/lib/WebSocket.js#L530

### CODE REVIEW

File: src/js/ripple/utils.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+var pJson = require('../../../package.json');
```
Comment (wltsmrz) [inline src/js/ripple/utils.js]: Wouldn't you rather call this `package`?

Comment (lumberj) [inline src/js/ripple/utils.js]: Copied from https://github.com/ripple/ripple-rest/blob/7142262ef0d1f4b367a37ffa2d5b8db7b56eb11b/api/lib/utils.js#L4

Comment (clark800) [inline src/js/ripple/utils.js]: > Copied from ...

That doesn't mean it's good

Comment (lumberj) [inline src/js/ripple/utils.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar

Future reserved words include `package`

**Comment by wltsmrz** [comment]
LGTM

File: src/js/ripple/utils.js (hunk 1 above)
Comment (clark800) [inline src/js/ripple/utils.js]: Why not call it packageJson? I think abbreviations are bad for readability.

Comment (lumberj) [inline src/js/ripple/utils.js]: Changed

**Comment by clark800** [comment]
LGTM

**Comment by clark800** [comment]
Can we add a test case?

**Pull request merged**